exercises/01_sort4/sort4.go

```diff
@@ -24,4 +27,5 @@ func main() {
 	a, b, c, d = sort4(a, b, c, d)
 
 	fmt.Println("Sorted numbers:", a, b, c, d)
-}
+} //input: 2 6 3 7 | -3 1 0 -6 | 0 1 0 1 | 38 951 73 253 |
```

Comment:
Now, when you know how to add tests for a program, you can modify `_test.go` directly and add those tests there.

Same applies for other programs.